sam-styles/packages/components/side-nav/styles/side-nav.scss

```diff
@@ -63,11 +63,18 @@
   
   
     .usa-sidenav__item{
-       @include u-border-top(0);
         @include u-border-x('1px');
         @include u-border-bottom('1px');
         @include u-border('base-light');
-  
+
+        &:first-child {
+            @include u-border-top('1px');
+        }
+        
+        &:not(:first-child) {
```

Comment:
Seems odd that you're only including just the first on line 70, but then having to ignoring the rest. Are you sure the :first-child is applying only to the first?

Reply:
Ah nevermind, it's a uswds style that is blanket adding border-top. So yeah you'd have to undo that for every other item.